styles/Parties.module.css

```
@@ -43,3 +43,22 @@
   margin: 0 auto 1rem;
   text-align: center;
 }
+
+.filters {
+  display: flex;
+  margin-bottom: 32px;
+}
+
+.filter {
+  width: 300px;
+  display: flex;
+  flex-direction: column;
+}
+
+.filter + .filter {
+  margin-left: 32px;
+}
+
+.filter label {
+  margin-bottom: 8px
+}
```

Creo que podríamos agregar algo como esto para mobile
```suggestion
}

@media (max-width: 600px) {
  .filters {
    flex-direction: column;
  }

  .filter + .filter {
    margin-left: 0;
    margin-top: 32px;
  }
}
```